variables.tf

```diff
@@ -44,3 +44,9 @@ variable "tags" {
   default     = {}
   type        = map(string)
 }
+
+variable "stream_mode_details" {
+  description = "Capacity mode of the stream, either ON_DEMAND or PROVISIONED"
```

Comment:
@jordan-tr do you think its worth adding a cost warning to users about using ON_DEMAND?  At a base level this setup will cost 2-3x more with much higher costs for data writes and reads as compared to PROVISIONED throughput?

Reply:
Yeah that might be a good idea as if you're not familiar with kinesis it would inflate their costs as you mentioned. The hope would be that if they were to change any settings they would first read up on it before doing it, but I could see how liability wise you would want to make sure your basis was covered

Reply:
Ill merge this into a release branch and a small note about cost.